pkg/runner/runner.go

```diff
@@ -160,7 +159,11 @@ func CreateClient(cfg *Config, promMetrics *api.Metrics) (*pgclient.Client, erro
 
 	migration_success := true
 	if cfg.Migrate {
-		err := migrate(&cfg.PgmodelCfg, appVersion, schemaVersionLease)
+		conn, err := schemaVersionLease.Conn()
```

Comment:
schemaVersionLease allowed to be nil here, so this doesn't work

Reply:
blah, forgot we allowed that 😞